index.js

```diff
@@ -2,3 +2,5 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 "use strict";
+
+module.exports.sign = require('./lib/sign').sign;
```

Comment:
I'd rather not include this unless you add test coverage. I still don't think it's necessary though. It's safe enough to `require('jpm/lib/sign')` in your script